final_task/rss_reader/classes/news_class.py

```diff
@@ -0,0 +1,39 @@
+"""Module contain definition of "News" class"""
+
+from dataclasses import dataclass
+
+@dataclass
+class News:
+    """Class that represent single entry of news
+
+    Uses for storing title, date, link, text and links from text
+    as an attributes of class.
+    """
+    title: str
+    date: str
+    link: str
+    text: str
+    links: str
+    feed_title: str
+    source: str
+
+
+    def print_feed_title(self):
+        """Print feed title"""
+        print(self.feed_title)
+
+
+    def print_news(self):
+        """Print to stdout title, date, link, text
+        and links from text in particular sequence."""
+        print('-----------------------------------------------------------')
+        print('Title: '+self.title)
+        print('Date: '+self.date)
+        print('Link: '+self.link)
+        print()
```

Comment:
It is better to think of some other solution for printing an empty line instead of using `print()`

Reply:
Fixed